bookkeeper-server/src/main/java/org/apache/bookkeeper/client/BKException.java

```diff
@@ -142,6 +144,7 @@ public static BKException create(int code) {
         int UnauthorizedAccessException = -102;
         int UnclosedFragmentException = -103;
         int WriteOnReadOnlyBookieException = -104;
+        int LedgerIdOverflowException = -106;
```

Comment:
What happened to -105?  Was it previously already used?  Or is it part of code Yahoo intends to push back?